src/components/StyleGuide/Filtering/JavaScriptDisabled.mdx

```diff
@@ -0,0 +1,29 @@
+import { Meta } from "@storybook/blocks";
+
+<Meta title="Style Guide/Filtering/JavaScript disabled" />
+
+# Filtering when JavaScript is disabled
```

Comment:
Once I have better progressive enhancement updates and recommendation, I want to update this to say that a fallback message banner should render in cases where features are JS-heavy.

Reply:
@EdwinGuzman Great. Pass it along when you are ready.